core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpExecutionServiceDAO.scala

```diff
@@ -81,6 +83,11 @@ class HttpExecutionServiceDAO( executionServiceURL: String, submissionTimeout: F
     retry(when500) { () => pipelineNoAuth[ExecutionServiceVersion] apply Get(url) }
   }
 
+  override def getStatus(): Future[Map[String, SubsystemStatus]] = {
+    val url = executionServiceURL + s"/engine/v1/status"
+    retry(when500) { () => pipelineNoAuth[Map[String, SubsystemStatus]] apply Get(url) }
```

Comment:
Is retrying on 500 errors appropriate here? Maybe we should just interpret a 500 response as a non-ok status.

Reply:
I say leave the retries, at least for now, so we can be resilient towards short lived glitches

Reply:
Agora status doesn't retry: https://github.com/broadinstitute/rawls/blob/develop/core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpMethodRepoDAO.scala#L85

Reply:
that's a good point - 500 is actually how it reports non-ok